### PR TITLE
[readme] fixed image and links

### PR DESCRIPTION
**Pull request by tanya-rai**
[readme] fixed image and links

fixed image sizing
added links that were previously missing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,10 +92,8 @@ AIConfigs are JSON-serializable configs that store your prompts, models, and mod
 </code></pre>
 </details>
 
-<br>
-
 The core SDK allows you to use your AIConfig easily in your application code.
-We cover Python instructions here, for Node.js please see the detailed Getting Started guide.
+We cover Python instructions here, for Node.js please see the detailed Getting Started guide [here](https://aiconfig.lastmileai.dev/docs/getting-started).
```
Comment (rholinshead) [inline README.md:96]:
```suggestion
We cover Python instructions here, for Node.js please see the detailed [Getting Started guide] (https://aiconfig.lastmileai.dev/docs/getting-started).
```
nit

File: README.md [hunk 2]
```diff
@@ -159,11 +157,11 @@ AIConfig helps unwind complexity by separating prompts, model parameters, and mo
 
 ## Features
 
-- **Prompts as Configs**: a [standardized JSON format](https://aiconfig.lastmileai.dev/docs/overview/ai-config-format) to store prompt chains and model settings in source control.
+- **Prompts as Configs**: [standardized JSON format](https://aiconfig.lastmileai.dev/docs/overview/ai-config-format) to store prompt and model settings in source control.
```
Comment (rholinshead) [inline README.md:160]: Should this be 'store prompts'? As-is, it sort of sounds like just storing prompt settings even though we store the literal prompts themselves

File: README.md [hunk 3]
```diff
@@ -159,11 +157,11 @@ AIConfig helps unwind complexity by separating prompts, model parameters, and mo
...
 - **Editor for Prompt Chains**: Prototype and iterate on your prompt chains and model settings in [AI Workbooks](https://lastmileai.dev/workbooks/clooqs3p200kkpe53u6n2rhr9).
-- **Model-agnostic SDK**: Access the latest models from OpenAI, Google, Meta, and HuggingFace - easily extensible to work with any generative AI model.
-- **Multimodal Capabilities**: Build multimodal AI apps that chain text, image, and audio models together.
-- **Improved Collaboration**: Different people can be responsible for prompt engineering and app development. AIConfig makes it easy to work together given the separation of concerns between generative AI settings and app code.
+- **Model-agnostic SDK**: Access the latest models from OpenAI, Google, Meta, and HuggingFace - easily support your own model.
```
Comment (rholinshead) [inline README.md:162]: nit, I kind of like the previous wording better. My thought reading this is that it only supports models from those mentioned or ones that I host somewhere

File: README.md [hunk 4]
```diff
@@ -189,11 +187,11 @@ AIConfig supports the following models out of the box. See examples:
 - [Gemini](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Gemini)
 - [LLaMA](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/llama)
 - [LLaMA Guard](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/LLaMA-Guard)
-- [Google PaLM models (PaLM chat)](<(https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Multi-LLM-Consistency)>)
+- [Google PaLM models (PaLM chat)](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Multi-LLM-Consistency)
 - [Hugging Face Text Generation Task models (Ex. Mistral-7B)](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/HuggingFace)
 
 If you need to use a model that isn't provided out of the box, you can implement a `ModelParser` for it.
-[Instructions on how to support a new model](https://aiconfig.lastmileai.dev/docs/extensibility#1-bring-your-own-model).
+See [instructions](https://aiconfig.lastmileai.dev/docs/extensibility#1-bring-your-own-model) on how to support a new model on AIConfig.
```
Comment (rholinshead) [inline README.md:194]:
```suggestion
See [instructions](https://aiconfig.lastmileai.dev/docs/extensibility#1-bring-your-own-model) on how to support a new model in AIConfig.
```
nit

**Review by rholinshead — APPROVED** [review]
Accepting to unblock, just a few nits

File: README.md [hunk 5]
```diff
@@ -159,11 +157,11 @@ AIConfig helps unwind complexity by separating prompts, model parameters, and mo
...
 - **Editor for Prompt Chains**: Prototype and iterate on your prompt chains and model settings in [AI Workbooks](https://lastmileai.dev/workbooks/clooqs3p200kkpe53u6n2rhr9).
-- **Model-agnostic SDK**: Access the latest models from OpenAI, Google, Meta, and HuggingFace - easily extensible to work with any generative AI model.
-- **Multimodal Capabilities**: Build multimodal AI apps that chain text, image, and audio models together.
-- **Improved Collaboration**: Different people can be responsible for prompt engineering and app development. AIConfig makes it easy to work together given the separation of concerns between generative AI settings and app code.
+- **Model-agnostic SDK**: Access the latest models from OpenAI, Google, Meta, and HuggingFace - easily support your own model.
+- **Multimodal Support**: Build multimodal AI apps that chain text, image, and audio models together.
+- **Improved Collaboration**: Different people can be responsible for prompt engineering vs app development. AIConfig makes it easy to work together given the separation of concerns between generative AI settings and app code.
```
Comment (saqadri) [inline README.md:164]:
```suggestion
- **Collaborative Development**: AIConfig enables different people to work on prompt engineering and app development, and collaborate together by sharing the `aiconfig` artifact.
```

**Review by saqadri — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tanya-rai) [reply README.md:96]: gunna keep it as is

**Pull request merged**